drivers/SmartThings/matter-switch/profiles/light-color-level-illuminance-motion-1000-15000k.yml

```diff
@@ -0,0 +1,27 @@
+name: light-color-level-illuminance-motion
```

Comment:
This name needs to be updated to match the file name.

Please update the filename and this profile `name:` to be `light-color-level-illuminance-motion-1000K-15000K` (capitalize the `K` and include it in after both numbers)

Reply:
Completed，please review.  Thanks！